release-notes/Web_apps/Web_apps_Feature_Release_10.4/Web_apps_Feature_Release_10.4.3.md

```diff
@@ -137,7 +137,13 @@ When, in the template editor, you pressed ESCAPE while the event editor window w
 
 <!-- MR 10.3.0 [CU12] - FR 10.4.3 -->
 
-A number of enum column issues have been fixed in the template editor.
+A number of enum column issues have been fixed in the Template Editor:
+
+- Query rows now include both display values and values. This change optimizes performance, especially for rows visible across multiple tables.
```

Comment:
Doesn't have anything to do with performance. It mostly prevents confusion when feeding a query row to other places. Previously the display values could not be used in eg: templates